components/VideoPlayer.vue

```diff
@@ -128,12 +141,87 @@ export default Vue.extend({
   beforeDestroy() {
     if (this.player) {
       window.muxjs = undefined;
+      this.onVideoStopped(); // Report that the playback is stopping
+      this.player.removeEventListener('error', this.onPlayerError);
       this.player.unload();
       this.player.destroy();
     }
   },
   methods: {
-    ...mapActions('snackbar', ['pushSnackbarMessage'])
+    ...mapActions('snackbar', ['pushSnackbarMessage']),
+    onVideoPlaying(_event: Event) {
+      // TODO: Move to playback manager
+      this.$api.playState.reportPlaybackStart({
+        playbackStartInfo: {
+          CanSeek: true,
+          ItemId: this.item.Id,
+          PlaySessionId: this.playbackInfo.PlaySessionId,
+          MediaSourceId: this.playbackInfo.MediaSources?.[0].Id,
+          AudioStreamIndex: 0, // TODO: Don't hardcode this
+          SubtitleStreamIndex: 0 // TODO: Don't hardcode this
+        }
+      });
+
+      this.lastProgressUpdate = new Date().getTime();
+    },
+    onVideoProgress(_event?: Event) {
+      // TODO: Move to playback manager
+      const now = new Date().getTime();
+
+      if (!(now - this.lastProgressUpdate < 700)) {
```

Comment:
This seems incorrect. Shouldn't we not report playback if the time interval is <700